Makefile

```diff
@@ -29,7 +29,7 @@ static:
 	./scripts/gobuild.sh
 
 gotest:
-	GOPATH="$(shell pwd)/ecs-init/Godeps/_workspace:$(GOPATH)" go test -short -v -cover ./...
+	GOPATH="$(shell pwd)/ecs-init/Godeps/_workspace:$(GOPATH)" go test -short -v -cover -timeout 10m ./...
```

Comment:
Can  you set this to something lower? I don't think it needs to be greater than `10s` even